test/functional/test_runner.py

```diff
@@ -59,6 +59,7 @@
     'wallet_basic.py',                          # ~ 1155 sec
     'wallet_backup.py',                         # ~ 459 sec
     'mining_pos_reorg.py',                      # ~ 305 sec
+    'wallet_hd.py',                             #
```

Comment:
please add the avg test time (and eventually resort the list) here. You can pick it from latest Travis run (205 sec)

Reply:
done